CHANGELOG.md

```diff
@@ -3,15 +3,26 @@
 ## Unreleased
 
 - Components
+    - General
+        - Remove YokeTraitHack from icu (unicode-org#6126)
```

Comment:
I think this is a very internal change that should be mentioned under utils/yoke only